FordLabs/PeopleMover
# Made location and product tag filtering an And instead of Or

## Issue
Connects #400 

## What was done
- [x] location and product filter are an AND instead of a OR
- [x] Ashley fix a rendering issue of the product list that was being render multiple times (may fix some flickering on loading space)

## How to test
1.create multiple products with different location and product tag and filter by location and product tag and make sure that only the product with the location and the product tag show up in the list
